packages/node-resolve/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@rollup/plugin-node-resolve",
+  "version": "6.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Locate and bundle third-party dependencies in node_modules",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Rich Harris <richard.a.harris@gmail.com>",
+  "homepage": "https://github.com/rollup/plugins/packages/node-resolve/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/index.js",
+  "engines": {
+    "node": ">= 8.0.0"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "ci:coverage": "nyc pnpm run test && nyc report --reporter=text-lcov > coverage.lcov",
+    "ci:lint": "pnpm run build && pnpm run lint && pnpm run security",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "ci:test": "pnpm run test -- --verbose && pnpm run test:ts",
+    "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
+    "lint:docs": "prettier --single-quote --write README.md",
+    "lint:js": "eslint --fix --cache src test",
+    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
+    "prebuild": "del-cli dist",
+    "prepare": "pnpm run build",
+    "prepublishOnly": "pnpm run lint && pnpm run test && pnpm run test:ts",
+    "pretest": "pnpm run build",
+    "security": "echo 'pnpm needs `npm audit` support'",
+    "test": "ava",
+    "test:ts": "tsc types/index.d.ts test/types.ts --noEmit"
+  },
+  "files": [
+    "dist",
+    "types",
+    "README.md",
+    "LICENSE"
+  ],
+  "keywords": [
+    "rollup",
+    "plugin",
+    "es2015",
+    "npm",
+    "modules"
+  ],
+  "peerDependencies": {
+    "rollup": "^1.20.0"
+  },
+  "dependencies": {
+    "@rollup/pluginutils": "^3.0.0",
+    "@types/resolve": "0.0.8",
+    "builtin-modules": "^3.1.0",
+    "is-module": "^1.0.0",
+    "resolve": "^1.11.1"
```

Comment:
Might be worthwhile to review which `dependencies` are being used, skimmed the code and `resolve` was the only one I was able to find. Shouldn't block the merge though, I can do it in a follow-up if it'll help

Reply:
I'm glad you brought this up. `src/index.js` is using `rollup-pluginutils` but the dependency is `@rollup/plugin-utils`. That's likely working due to resolution with pnpm, but it's not right. Fixed.

Reply:
Thanks for pointing that out, looks like GitHub automatically collapsed src/index.js because it was a large diff. I see the rest of the deps used now 👍